USAGE.md

```diff
@@ -11,36 +11,61 @@ The keywords "must", "must not", "required", "shall", "shall not", "should", "sh
 
 ## Geostore account and resource names
 
-Following information must be provided by Geostore instance maintainer in order to start using it:
+You'll need the following information to start using Geostore:
 
 - Geostore AWS account ID (`GEOSTORE_AWS_ACCOUNT_ID`)
 - Geostore user role name (`GEOSTORE_USER_ROLE_NAME`)
+- If you do not already have a bucket which contains your datasets, you can use the Geostore staging
+  bucket (`GEOSTORE_STAGING_BUCKET_NAME`) and related user role (`GEOSTORE_STAGING_USER_ROLE_NAME`)
+
+Replace the `GEOSTORE_…` strings in the following documentation with the actual values given by the
+Geostore instance maintainer.
 
 ## Dataset source S3 bucket
 
-To import data into Geostore, the dataset source S3 bucket must be readable by Geostore.
+Geostore needs to be able to read all the files which comprise the dataset in the source bucket.
```

Comment:
Good point; I added it because I thought I remembered something about some users using our bucket since they wouldn't have  one of their own. @billgeo?

Reply:
Yes, we may implement something like this later, but for now should just remove this from the docs.

Reply:
All done, cheers!